locales/en-US/editor.properties

```diff
@@ -208,6 +208,24 @@ PDF_FILE_TITLE=PDF File
 ################
 ## EXTENSIONS ##
 ################
+# extensions/default/InlineBorderRadiusEditor
+ALL_CORNERS=All Corners
+INDIVIDUAL_CORNERS=Individual Corners
+TL_RADIO_PX=pixels
```

Comment:
All these strings are the same. Just use `PIXEL_UNIT=pixels`, `EM_UNIT=em font unit`, `PERCENTAGE_UNIT=percentage`